docs/static/include/pluginbody.asciidoc

```diff
@@ -275,40 +275,29 @@ require "logstash/namespace"
 class LogStash::{pluginclass}::{pluginnamecap} < LogStash::{pluginclass}::Base
   config_name "example"
 
-  # If declared logstash will only allow a single instance of this plugin
-  # to exist, regardless of how many CPU cores logstash detects. This is best
-  # used in cases like the File output, where separate threads writing to a single
-  # File would only cause problems.
+  # This is the default concurrency level of this output plugin.
+  # You do not need to explicitly set it.
+  # When configured as :single a single instance of the Output will be shared among the
+  # pipeline threads. Access to the `#multi_receive/#receive` method will be synchronized
```

Comment:
How bout I rename it pipeline worker threads. Since we're talking about synchronization I like that we're informing the reader that these are os threads.